spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java

```diff
@@ -1169,7 +1169,7 @@ public static void validatePartitionFilter(
       Preconditions.checkArgument(
           unMatchedFilters.isEmpty(),
           "Cannot add files to target table %s. %s is partitioned but the specified partition filter "
-              + "refers to columns that are not partitioned: '%s' . Valid partition columns %s",
+              + "refers to columns that are not partitioned: '%s' . Valid partition columns: '%s'",
```

Comment:
`'...'`  style looks weird to me when it takes several column names. I would use `[...]` or add only `:`.

Reply:
done